Cargo.toml

```diff
@@ -27,3 +27,6 @@ mmio = ["vmm/mmio_support"]
 integration_tests = []
 
 [workspace]
+members = [
+    "vhost_user_backend",
```

Comment:
Not sure about this? I think we should have them all or nothing.

Reply:
Yes that was the wrong way to do it. I've just fixed by making the crate its own workspace.